include/triton/backend/backend_common.h

```diff
@@ -353,6 +353,34 @@ TRITONSERVER_Error* ReadInputTensor(
     TRITONBACKEND_Request* request, const std::string& input_name, char* buffer,
     size_t* buffer_byte_size, const char* host_policy_name = nullptr);
 
+/// Get an input tensor's contents into a buffer.
```

Comment:
Can you add a line here on when to use this signature vs the previous one. There are a lot of parameters in these functions and the difference might not be evident just after a cursory look.